wurst/data/HashMap.wurst

```diff
@@ -29,6 +29,28 @@ public class HashMap<K,V> extends Table
 		remove(key)
 		return result
 
+public class SizedMap<K,V> extends HashMap<K,V>
```

Comment:
SizedMap kinda gives the illusion that the Map is constrained by a size, rather than just counting imo. @cokemonkey11 ?

Reply:
I guess a size-constrained one might be called "SmallMap" or something. SizedMap feels to me like `Map<> implements Sized` so name feels ok

I don't see the point though in extending hashmap. Why not just add size to hashmap?

Reply:
guess the only argument is the introduced overhead.
If you think it doesn't matter, we should just add it to HashMap.

Reply:
`guess the only argument is the introduced overhead.`
i think add more overhead is unnecessary (with only a feature )

Reply:
Instead of adding this to the HashMap, why not just use the `IterableMap`? It already has the `size` method with minimal overhead (memory increase based on the number of keys). 

If we still want size for the a standard HashMap, I would prefer it goes into the standard HashMap. `size` methods are not uncommon methods for HashTable data structures and it would make sense for the Wurst implementation to have one.

Reply:
because the `IterableMap` has a inner `hashlist` .
if u think `adding this to the HashMap` is good. i can do it